packages/salesforce-adapter/src/filters/custom_objects_instances.ts

```diff
@@ -0,0 +1,134 @@
+/*
+*                      Copyright 2020 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+import _ from 'lodash'
+import { InstanceElement, ObjectType, Element, isObjectType, Field, ElemID, Values } from '@salto-io/adapter-api'
+import { SalesforceClient } from 'index'
+import { SALESFORCE, RECORDS_PATH, INSTALLED_PACKAGES_PATH } from '../constants'
+import { FilterCreator } from '../filter'
+import { apiName, isCustomObject, Types } from '../transformers/transformer'
+import { getNamespace } from './utils'
+
+const buildQueryString = (type: ObjectType): string => {
+  const selectStr = Object.values(type.fields)
+    .map((field: Field) => {
```

Comment:
nit: type can be inferred
```suggestion
    .map(field => {
```